src/components/SuggestionsList/SuggestionsList.tsx

```diff
@@ -8,12 +8,13 @@ import Text, { TextSize, TextColor } from '../Text';
 import Callout, { DirectionalHint } from '../Callout';
 import Block, { GutterSize } from '../Block';
 import './SuggestionsList.css';
+import { IPoint } from 'office-ui-fabric-react/lib/utilities/positioning';
 
 export interface SuggestionsListProps extends NestableBaseComponentProps {
   /**
    * The target that the Callout should try to position itself based on.
    */
-  target: HTMLElement;
+  target?:  HTMLElement | string | MouseEvent | IPoint | null;
```

Comment:
Why is it optional and `null`? Where will it render when it's `undefined` or `null`?

Reply:
Just aligned it to Fabrics. The only one I intend to use is IPoint

Reply:
Right, but what's the effect of `undefined` and `null`? If it doesn't make sense in our scenario, no sense in aligning.

Reply:
ok done